pkg/coscheduling/coscheduling.go

```diff
@@ -94,11 +100,22 @@ func (cs *Coscheduling) Name() string {
 }
 
 // New initializes a new plugin and returns it.
-func New(_ *runtime.Unknown, handle framework.FrameworkHandle) (framework.Plugin, error) {
+func New(arg *runtime.Unknown, handle framework.FrameworkHandle) (framework.Plugin, error) {
+	var args Args
+
+	var params []byte
+	if arg != nil {
+		params = arg.Raw
+	}
```

Comment:
FYI: https://github.com/kubernetes/kubernetes/blob/v1.17.11/pkg/scheduler/framework/plugins/requestedtocapacityratio/requested_to_capacity_ratio.go#L40

Reply:
> FYI: https://github.com/kubernetes/kubernetes/blob/v1.17.11/pkg/scheduler/framework/plugins/requestedtocapacityratio/requested_to_capacity_ratio.go#L40

Again, I can simplify the implementation using framework.DecodeInto, but don't know how to unmarshal to time.Duration directly.

Reply:
Thanks, made the changes using DecodeInto.